tests/backends/postgresql/test_creation.py

```diff
@@ -61,6 +62,15 @@ def test_sql_table_creation_suffix_with_encoding_and_template(self):
         settings = {'CHARSET': 'UTF8', 'TEMPLATE': 'template0'}
         self.check_sql_table_creation_suffix(settings, '''WITH ENCODING 'UTF8' TEMPLATE "template0"''')
 
+    def test_sql_table_creation_raises_improperly_configured_error_when_collation_in_settings(self):
+        settings = {'COLLATION': 'test', 'CHARSET': 'UTF8', 'TEMPLATE': 'template0'}
+        message = "PostgreSQL does not support collation setting at " \
+                  "database creation time."
+        with self.assertRaisesMessage(ImproperlyConfigured, message):
+            self.check_sql_table_creation_suffix(settings,
+                                                 '''WITH ENCODING 'UTF8' TEMPLATE "template0"
+                                                 AND COLLATION''')
```

Comment:
`sql_table_creation_suffix()` will raise an exception so there is no need to pass anything except `COLLATION`.

Reply:
Great, thanks! Have a nice day 😄